test/functional/test_runner.py

```diff
@@ -269,9 +269,20 @@ def main():
     test_list = []
     if tests:
         # Individual tests have been specified. Run specified tests that exist
-        # in the ALL_SCRIPTS list. Accept the name with or without .py extension.
+        # in the ALL_SCRIPTS list. Accept names with or without a .py extension.
+        # Specified tests can contain paths and wildcards, but the paths must be
+        # coherent, e.g. the same path as that provided to call test_runner.py.
+        # Examples:
+        #   `test/functional/test_runner.py test/functional/wallet*`
+        #   `test/functional/test_runner.py ./test/functional/wallet*`
+        #   `test_runner.py wallet*`
+        #   but not:
+        #   `test/functional/test_runner.py wallet*`
+        # Multiple wildcards can be passed:
+        #   `test_runner.py tool* mempool*`
```

Comment:
Sorry for the follow up comment, but the test runner only sees a list of file names and not the wildcard, so mentioning it here is a bit confusing. Also, incoherent paths are accepted. You can pass e.g. `test/functional/test_runner.py /asdf/wallet_disable.py  /foo2/wallet_disable wallet_disable`.

Maybe remove this here? (At most you could mention that path prefixes are stripped in the argparse doc of this script)

Reply:
No problem. True. Though with wildcards those incoherent paths don't seem to be accepted. I'll rework this.

Reply:
Reworded:
```
--- Specified tests can contain paths and wildcards, but the paths must be coherent
+++ Specified tests can contain wildcards, but in that case the supplied paths should be coherent
```
I'm happy to remove the added comments if they are still confusing.